test/queries.test.js

```diff
@@ -1,23 +1,23 @@
-const { assert } = require('chai');
```

Comment:
it looks like prettier has converted all single quotes to double, @Dannyqu can you move the prettier.rc file from ./src into the root directory and run prettier again on this file